src/main/java/org/betonquest/betonquest/compatibility/protocollib/wrappers/WrapperPlayClientSteerVehicleUpdated.java

```diff
@@ -0,0 +1,88 @@
+package org.betonquest.betonquest.compatibility.protocollib.wrappers;
+
+import com.comphenix.packetwrapper.WrapperPlayClientSteerVehicle;
+import com.comphenix.protocol.events.PacketContainer;
+import com.comphenix.protocol.reflect.StructureModifier;
+
+/**
+ * Wrapper for the PacketPlayClientSteerVehicle class.
+ */
+public class WrapperPlayClientSteerVehicleUpdated extends WrapperPlayClientSteerVehicle {
+
+    /**
+     * Exception message for not implemented methods.
+     */
+    private final static String NOT_IMPLEMENTED = "Not implemented";
+
+    /**
+     * Default constructor.
+     */
+    public WrapperPlayClientSteerVehicleUpdated() {
+        super();
+
+    }
+
+    /**
+     * @param packet - the packet to wrap.
+     */
+    public WrapperPlayClientSteerVehicleUpdated(final PacketContainer packet) {
+        super(packet);
+    }
+
+    @Override
+    public float getSideways() {
+        final StructureModifier<Boolean> input = this.getHandle().getStructures().read(0).getBooleans();
+
+        float sideways = 0;
+
+        if(input.read(2)) {
+            sideways += 1;
+        }
+        if(input.read(3)) {
+            sideways -= 1;
+        }
+
+        return sideways;
+    }
+    @Override
+    public void setSideways(final float value) {
+        throw new UnsupportedOperationException("Not implemented");
+    }
+    @Override
+    public float getForward() {
+        final StructureModifier<Boolean> input = this.getHandle().getStructures().read(0).getBooleans();
+
+        float forward = 0;
+
+        if(input.read(0)) {
+            forward += 1;
+        }
+        if(input.read(1)) {
+            forward -= 1;
+        }
+
+        return forward;
+
+    }
+    @Override
+    public void setForward(final float value) {
+        throw new UnsupportedOperationException(NOT_IMPLEMENTED);
+    }
+    @Override
+    public boolean isJump() {
+        return this.handle.getStructures().read(0).getBooleans().read(4);
+    }
+    @Override
+    public void setJump(final boolean value) {
+        throw new UnsupportedOperationException(NOT_IMPLEMENTED);
+    }
+    @Override
+    public boolean isUnmount() {
+        return this.handle.getStructures().read(0).getBooleans().read(5);
+    }
+    @Override
+    public void setUnmount(final boolean value) {
+        throw new UnsupportedOperationException(NOT_IMPLEMENTED);
+    }
```

Comment:
Why dont implement the missing methods, as its a real simple set? Or I am wrong?

Reply:
Didnt implement them because they are not used

Some would be simple sets, the setForward  / setUnmount would require additional logic